src/sonic-host-services/scripts/hostcfgd

```diff
@@ -8,8 +8,8 @@ import subprocess
 import syslog
 
 import jinja2
-from sonic_py_common import device_info
-from swsssdk import ConfigDBConnector
+from sonic_py_common import device_info,multi_asic
+from swsssdk import ConfigDBConnector, SonicDBConfig
```

Comment:
for master we no longer use swsssdk, it is depracating. @qiluo-msft

Reply:
@qiluo-msft , are you going to refactor this?

Reply:
@arlakshm Could you just replace `swsssdk` with `swsscommon.swsscommon` ? we are deprecating `swsssdk`

Reply:
change in latest commit